integration_testing/features/drag-drop-content-nodes-into-the-floating-action-button.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Drag-drop content nodes into the floating action button

```

Comment:
Unify this and the previous in the single `.feature` file named `Drag-drop to clipboard` with 2 scenarios.